src/main/java/codeu/model/data/User.java

```diff
@@ -31,12 +32,28 @@
    * @param name the username of this User
    * @param password the password of this User
    * @param creation the creation time of this User
+   * @param aboutMe the about me description of this User
    */
-  public User(UUID id, String name, String password, Instant creation) {
+
+  public User(UUID id, String name, String password, Instant creation, String aboutMe) {
```

Comment:
Can you add a test in UserTest.java for creating a user with aboutMe set? Also update the current test to make sure the default is set correctly.